xbmc/pvr/addons/PVRClients.cpp

```diff
@@ -1566,6 +1567,15 @@ bool CPVRClients::OpenStream(const CPVRRecordingPtr &channel)
       client->OpenStream(channel))
   {
     CSingleLock lock(m_critSection);
+
+    CDateTime endTime = channel->RecordingTimeAsLocalTime() +
+        CDateTimeSpan(0, 0, channel->GetDuration() / 60, channel->GetDuration() % 60);
+
+    m_bIsRecordingInProgress = (endTime > CDateTime::GetCurrentDateTime());
```

Comment:
It's no problem to change my PR to use UTC times, but then we should of course change the doxy as @metaron-uk suggests.

@FernetMenta Why could we not use buffering for all pvr related things (live and recording)?
People might watch recordings over the internet as well (like I do) and then they will benefit from this too?